### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by jnation3406**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,19 +4,22 @@
 [![Coverage Status](https://coveralls.io/repos/github/observatorycontrolsystem/ingester/badge.svg?branch=master)](https://coveralls.io/github/observatorycontrolsystem/ingester?branch=master)
 [![Codacy Badge](https://app.codacy.com/project/badge/Grade/24eb8debeb0c499ca192b4497a1f1e12)](https://www.codacy.com/gh/observatorycontrolsystem/ingester?utm_source=github.com&utm_medium=referral&utm_content=observatorycontrolsystem/ingester&utm_campaign=Badge_Grade)
 
-A library for adding new science data products to an observatory control system's science archive. The library
-handles uploading files into AWS S3, as well as adding records to the science archive's database containing
-the searchable metadata of all available files. It records metrics in an openTSDB about ingestion.
+A library for adding new science data products to an observatory control system's [Science Archive](https://github.com/observatorycontrolsystem/science-archive/).
+The library handles uploading files into a `FileStore` (usually S3), as well as adding records to the Science Archive's database containing
```
Comment (mgdaily) [inline README.md:8]: s3 recommended?

File: README.md [hunk 2]
```diff
@@ -4,19 +4,22 @@
...
+the searchable metadata of all available files. It records ingestion metrics in an openTSDB instance, which can be disabled by setting the environment variable `OPENTSDB_PYTHON_METRICS_TEST_MODE=False`.
```
Comment (mgdaily) [inline README.md:9]: **Optionally** it records ingestion metrics in an [OpenTSDB](http://opentsdb.net/) (add a link) instance.

File: README.md [hunk 3]
```diff
@@ -4,19 +4,22 @@
...
 ## Prerequisites
 
 Optional prerequisites may be skipped for reduced functionality.
 
--   Python >= 3.6
--   A running [science archive](https://github.com/observatorycontrolsystem/science-archive)
--   Write access to the same S3 bucket that the running science archive is using
+-   Python >= 3.7
+-   A running [Science Archive](https://github.com/observatorycontrolsystem/science-archive/)
+-   A configured FileStore (usually S3) with write access to store data there
```
Comment (mgdaily) [inline README.md:17]: s3 recommended?

**Review by mgdaily — APPROVED** [review]

**Comment by mgdaily** [comment]
Oh one more thing I had on my list - PTR ran into an issue uploading data products to their s3 bucket because the `FILESTORE_TYPE` wasn't listed as a required env var in the ingester readme and they never set it, so it failed to upload to s3 silently. Can we add that?

**Comment by jnation3406** [comment]
> 

Do you mean add an explicit README comment about the filestore type var? I do link to the ocs archive library readme and say to look through all those env variables, but maybe it is worth calling out again.

**Comment by mgdaily** [comment]
> > 
> 
> Do you mean add an explicit README comment about the filestore type var? I do link to the ocs archive library readme and say to look through all those env variables, but maybe it is worth calling out again.

I was just thinking of adding that to the table under the "ingester" section. Guess it wasn't obvious to the PTR folk that they were supposed to set that variable, and I remember it being crucial to the pipelines too. Maybe emphasizing it again that way would work.

**Comment by jnation3406** [comment]
> > > 
> > 
> > 
> > Do you mean add an explicit README comment about the filestore type var? I do link to the ocs archive library readme and say to look through all those env variables, but maybe it is worth calling out again.
> 
> I was just thinking of adding that to the table under the "ingester" section. Guess it wasn't obvious to the PTR folk that they were supposed to set that variable, and I remember it being crucial to the pipelines too - basically any user of the ingester. Maybe emphasizing it again that way would work.

Okay Ill add some text about it. I wanted to avoid copying the entire set of environment variables from the ocs_archive project readme into the ingester and science archive readmes to, but I understand some users won't take the time to read both.

**Pull request merged**